src/main/java/net/mcreator/ui/modgui/ModElementGUI.java

```diff
@@ -571,7 +571,8 @@ protected void afterGeneratableElementGenerated() {
 	}
 
 	protected boolean allowCodePreview() {
```

Comment:
Is this change still needed?

Reply:
Now this passes the `GeneratableElement` from data provided by the `ModElementGUI`, in case null it gets replaced with the cached element version. Does this seem to not do a thing?